src/util/dev/server.ts

```diff
@@ -490,14 +490,12 @@ export default class DevServer {
       }
     );
     if (needsInitialBuild.length > 0) {
-      const stopSpinner =  wait('Building...');
+      this.output.log('Creating initial build');
```

Comment:
Maybe we should make this `builds` instead of `build` when there is more than one build?

Reply:
We have a plural function that can take care of that

Reply:
@AndyBitz Thank! I updated it.